### DISCUSSION TITLE
Clear stage to black & Mouse scroll wheel input?

### DISCUSSION THREAD

**Discussion by ghost** [AlmasB/FXGL#1279] [Q&A] [answered]
Hi all, I've just started playing around and found most of my beginner questions answered in the docs, except:

- It seems by default the background colour is white/light grey. is there a sort of glClearColor in FXGL/JavaFX. I was planning on having a scrolling background image in my game, but for now, it would be nice if the background was black.
- I want to use the mouse wheel to zoom in/out and couldn't find what I needed in the Input stuff, so came up with this which seems to work, but is there a better/proper way?
```
FXGL.getPrimaryStage().getScene().setOnScroll(event -> {
      if(event.getDeltaY()>0) {
        vp.setZoom(vp.getZoom()+(tpf *2f));
      } else if(event.getDeltaY()<0) {
        vp.setZoom(vp.getZoom()-(tpf *2f));
      }
    });
(vp is a reference to the Viewport)
```
(I don't have a 'player' as such, just moving the ViewPort around the play area.)

**Comment by AlmasB** [comment, marked as answer]
Hi there,

1) `FXGL.getGameScene().setBackgroundColor(Color.BLACK)`. Here is a [sample](https://github.com/AlmasB/FXGL/blob/67e7db4db16e11ccb19f3db4494e7f27248df003/fxgl-samples/src/main/java/intermediate/ScrollingBackgroundSample.java) showing how to use a scrolling background for when you need it.

2) This is fine as there is no built-in way of listening for zoom events in Input.

Feel free to post any questions that may arise during development.

**Reply by ghost** [reply]
Thank you so much (for the answers and the game engine!)
